rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/IBaseTextDocumentService.java

```diff
@@ -44,5 +44,5 @@ public interface IBaseTextDocumentService extends TextDocumentService {
     void unregisterLanguage(LanguageParameter lang);
     CompletableFuture<IValue> executeCommand(String languageName, String command);
     LineColumnOffsetMap getColumnMap(ISourceLocation file);
-
+    String getContents(ISourceLocation file);
```

Comment:
The reason I opted for this signature is because both implementing classes already had a function like this. But after some refactorings, this is not required anymore, so I'll revert to how it was.